common/src/main/java/com/microsoft/identity/common/internal/ui/webview/OAuth2WebViewClient.java

```diff
@@ -185,10 +185,14 @@ private void checkStartUrl(final String url) {
             Logger.info(TAG, "onPageStarted: Non-hierarchical loading uri.");
             Logger.infoPII(TAG, "start url: " + url);
         } else if (StringUtil.isEmpty(uri.getQueryParameter(AuthenticationConstants.OAuth2.CODE))) {
-            Logger.infoPII(TAG, "Host: " + uri.getHost() + " Path: " + uri.getPath());
+            Logger.info(TAG, "onPageStarted: URI has no auth code ('"
+                    + AuthenticationConstants.OAuth2.CODE + "') query parameter.");
+            Logger.infoPII(TAG, "Scheme:" + uri.getScheme() + " Host: " + uri.getHost()
+                    + " Path: " + uri.getPath());
         } else {
             Logger.info(TAG, "Auth code is returned for the loading url.");
-            Logger.infoPII(TAG, "Host: " + uri.getHost() + " Path: " + uri.getPath());
+            Logger.infoPII(TAG, "Scheme:" + uri.getScheme() + " Host: " + uri.getHost()
+                    + " Path: " + uri.getPath());
         }
     }
 }
```

Comment:
Let's add a new line at the end of this file

Reply:
Added.